doc/conf.py

```diff
@@ -36,7 +36,9 @@
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
     'sphinx_tabs.tabs',
-    'IPython.sphinxext.ipython_console_highlighting'
+    'IPython.sphinxext.ipython_console_highlighting',
+    'sphinx.ext.autodoc',
```

Comment:
Already imported at line 36

Reply:
Duplicate removed in https://github.com/UCL-INGI/INGInious/pull/756/commits/9033abcd70f990f1379ce5666df725ca5e0bca0e.